canonical/canonical.com
# Data bubble migration

## Done

Changes related to data bubble

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Navigate to `/data` URL

## Issue / Card

This is a feature branch that accumulates several different tickets related to data bubble epic: https://warthogs.atlassian.net/browse/WD-3805
